utils/README.md

```diff
@@ -0,0 +1,37 @@
+# Running Tests (NEW)
+
+Run the command:
+
+    make ci
+
+to run an array of unittests locally.
+
+# Running From Source
+
+You will need to setup a **virtualenv** to run from source:
+
+    $ virtualenv oo-install
+    $ source ./oo-install/bin/activate
+    $ virtualenv --relocatable ./oo-install/
+    $ python setup.py install
```

Comment:
develop makes it easier to consume changes without having to 're-install': http://setuptools.readthedocs.io/en/latest/setuptools.html#development-mode